tools/wptserve/wptserve/pipes.py

```diff
@@ -471,21 +471,24 @@ def config_replacement(match):
             raise Exception("Undefined template variable %s" % field)
 
         while tokens:
-            ttype, field = tokens.popleft()
+            ttype, ffield = tokens.popleft()
```

Comment:
The `ttype` convention is a bit odd but this follows it at first glance. But it doesn't seem like it's to avoid shadowing a `type` variable, so I take it it's actually t as in token. Going by that, let me suggest `tfield` here instead.